checks/apps/vasp/src/makefile.include.gh200

```diff
@@ -0,0 +1,122 @@
+# Default precompiler options
```

Comment:
would it be possible to add a README with some info about the input files (or is that private info) ?

Reply:
VASP comes with a set of makefiles for different architectures. This file is just taken from there and slightly modified to include the correct link paths. The other input files are unchanged and originate from https://www.nsc.liu.se/~pla/vasptest, as commented in the `INCAR` file. Not sure if that's enough to put into a readme.